components/mail/postfix/Makefile

```diff
@@ -38,11 +40,13 @@ CCARGS+=	-DDEF_NEWALIAS_PATH=\\\"/usr/sbin/newaliases\\\"
 CCARGS+=	-DUSE_TLS
 CCARGS+=	-DHAS_DB
 CCARGS+=	-DHAS_SQLITE
+CCARGS+=	-DUSE_SASL_AUTH -DUSE_CYRUS_SASL -I/usr/include/sasl
 
 AUXLIBS=	-lssl
 AUXLIBS+=	-lcrypto
 AUXLIBS+=	-ldb-5.3
 AUXLIBS+=	-lsqlite3
+AUXLIBS+=	-L/usr/lib/sasl -lsasl
```

Comment:
do we need corresponding -R flag?

Reply:
Not as far as I know. CentOS/Fedora does not have a -R Flag. AFAIK many linuxes don't

Reply:
Does it work without -L ?

Reply:
Yep it works without -L. Don't know why the linuxes include it in that case good catch.